pkgs/by-name/re/re2/package.nix

```diff
@@ -44,7 +44,10 @@ stdenv.mkDerivation (finalAttrs: {
   propagatedBuildInputs = [ abseil-cpp ] ++ lib.optionals (!stdenv.hostPlatform.isStatic) [ icu ];
 
   cmakeFlags =
-    [ (lib.cmakeBool "RE2_BUILD_TESTING" true) ]
+    [
+      (lib.cmakeBool "RE2_BUILD_TESTING" true)
+      (lib.cmakeFeature "CMAKE_CTEST_ARGUMENTS" "--timeout;999999")
```

Comment:
I'm not a huge fan of this but I see that early timeouts seem to effect multiple users. Can we use a more reasonable limit here perhaps? How long do the tests run on your machine?

Reply:
Up to 1200 seconds. I think the default timeout was 600?

Reply:
I guess if its common then sure